src/lib/AnnotationControls.ts

```diff
@@ -84,21 +84,53 @@ export default class AnnotationControls {
     private handleFullscreenExit = (): void => this.handleFullscreenChange(false);
 
     /**
-     * Region comment button click handler
+     * Deactive current control button
      */
-    private handleRegionClick = (onRegionClick: RegionHandler) => (event: MouseEvent): void => {
+    public deactivateCurrentControl = (): void => {
+        if (!this.currentActiveControl) {
+            return;
+        }
+
+        switch (this.currentActiveControl) {
```

Comment:
I wonder if the switch is needed at all. Should it just blindly run through all the `update*Button` methods each time? For instance if the click region and then click highlight, we would need region and highlight to update respectively

Reply:
> I wonder if the switch is needed at all. Should it just blindly run through all the `update*Button` methods each time? For instance if the click region and then click highlight, we would need region and highlight to update respectively

This is `deactivateCurrentControl()`, which only deactivate the current control. Update next control will happen in `handle*Click`.